src/components/Pagination/styled.tsx

```diff
@@ -1,8 +1,10 @@
 import styled from 'styled-components'
 import LeftBlack from '../../assets/pagination_black_left.png'
 import LeftWhite from '../../assets/pagination_white_left.png'
+import LeftGreen from '../../assets/pagination_green_left.png'
 import RightBlack from '../../assets/pagination_black_right.png'
 import RightWhite from '../../assets/pagination_white_right.png'
+import RightGreen from '../../assets/pagination_green_right.png'
```

Comment:
`LeftBlack/LeftWhite/RightBlack/RightWhite` should be removed ?

Reply:
They are used on PC. `RightGreen  LeftGreen` only used on mobile.  
PC and Mobile have different design.

Reply:
OK